var/spack/repos/builtin/packages/mpibind/package.py

```diff
@@ -13,39 +13,48 @@ class Mpibind(AutotoolsPackage):
     to heterogeneous architectures"""
 
     homepage    = "https://github.com/LLNL/mpibind"
-    url         = "https://github.com/LLNL/mpibind/archive/refs/tags/v0.5.0.tar.gz"
     git         = "https://github.com/LLNL/mpibind.git"
 
     maintainers = ['eleon']
 
-    # The build process uses 'git describe --tags' to get the
-    # package version, thus we need 'get_full_repo'
+    # This package uses 'git describe --tags' to get the
+    # package version in Autotools' AC_INIT, thus
+    # 'get_full_repo' is needed.
+    # Furthermore, the package can't be cached because
+    # AC_INIT would be missing the version argument,
+    # which is derived with git.
     version('master', branch='master', get_full_repo=True)
-    version('0.7.0', sha256='33077e7eb50322d2bcfe87bb3ea9159c2e49f6f045cbbcd2e69e763c3bec4330')
-    version('0.5.0', sha256='51bb27341109aeef121a8630bd56f5551c70ebfd337a459fb70ef9015d97d2b7')
+    version('0.8.0',  tag='v0.8.0',    no_cache=True)
+    version('0.7.0',  tag='v0.7.0',    no_cache=True)
+    version('0.5.0',  tag='v0.5.0',    no_cache=True)
```

Comment:
Can we pin fixed versions to a commit sha?

Reply:
Greetings @alalazo. Are you asking to add a `commit` to the version statements above? Or replace `tag` with `commit`?

Reply:
Replace tag with commit, if possible. I'm asking since I assume version tags will not move in time, so a commit sha seems more appropriate.

Reply:
Done.

Reply:
@alalazo , anything else I need to do?